recipe/meta.yaml

```diff
@@ -34,14 +36,17 @@ requirements:
     - numpy >=1.12  # [win32 and py36]
     - scipy >=0.17
     - matplotlib >=1.3.1
-    - networkx >=1.8
+    - networkx >=1.8,<2.0
     - pillow >=2.1.0
     - dask-core >=0.15
     - toolz >=0.7.4
     - pywavelets >=0.4.0
     - imageio >=2.1.0
 
 test:
+  requires:
+    - pillow <4.2
```

Comment:
Should there be a lower bound too?

Reply:
Does the lower bound not propagate from the runtime requirements...?

Reply:
Good point. Overlooked that. Should be fine.